### ISSUE TITLE
BenchmarkRunner not working from VSTS agent

### ISSUE THREAD

**Issue by lovettchris** [dotnet/BenchmarkDotNet#1480]
I have a self hosted azure devops build agent so I can run perf tests on a dedicated machine (running as a network service), but when I run the job it fails to find any benchmarks.  

```
// **************************
// Benchmark: CreationThroughputBenchmark.MeasureCreationThroughput: DefaultJob [NumMachines=10000, DoHalt=False]
// *** Execute ***
// Launch: 1 / 1
// Execute: D:\VSTS\agent\_work\1\s\Tools\BenchmarkRunner\bin\net48\88e5f964-b399-4348-9f81-cc987e2f6d63.exe --benchmarkName "Microsoft.Coyote.Performance.Tests.Actors.StateMachines.CreationThroughputBenchmark.MeasureCreationThroughput(NumMachines: 10000, DoHalt: False)" --job "Default" --benchmarkId 0 in 
// BeforeAnythingElse
ExitCode != 0
// Benchmark Process 168 has exited with code -532462766
No more Benchmark runs will be launched as NO measurements were obtained from the previous run!
```

when I run my BenchmarkRunner.exe manually it works fine, on the same machine, any ideas?  When I run DefaultJob.bat I get these interesting compile errors:

DefaultJob.notcs(43,47): error CS1069: The type name 'Enumerable' could not be found in the namespace 'System.Linq'. This type has been forwarded to assembly 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' Consider adding a reference to that assembly.

**Comment by adamsitnik** [comment]
Hi @lovettchris 

> when I run my BenchmarkRunner.exe manually it works fine, on the same machine,

that's surprising. Did you perform a full run, or have you stopped when it had printed `// BeforeAnythingElse` message?

Anytime BDN prints things like `// BeforeX` after `// AfterX` it expects the host process to read the output and write the word "Acknowledgment" to the output. Typically all the bugs happen after "// BeforeAnythingElse" so it's important that you put "Acknowledgment" and hit enter so it continues.

If you need some help with troubleshooting you can schedule a Teams meeting with me (my MS alias is `adsitnik`) but since I work remotely from Europe please schedule a meeting before 11 AM Redmond time.

Thanks,
Adam

**Comment by adamsitnik** [comment]
Fixed by #1491
